format/leveldb/testdata/ldb_uncompressed.fqtest

```diff
@@ -0,0 +1,98 @@
+*    |until 0x5f3.7 (118)                            |                |
+     |                                               |                |        trailer{}: 0x5f4-0x5fc (8)
+     |                                               |                |          restarts[0:1]: 0x5f4-0x5f8 (4)
+0x5f0|            00 00 00 00                        |    ....        |            [0]: 0 restart 0x5f4-0x5f8 (4)
+0x5f0|                        01 00 00 00            |        ....    |          num_restarts: 1 0x5f8-0x5fc (4)
+0x5f0|                                    00         |            .   |      compression: "none" (0x0) 0x5fc-0x5fd (1)
+0x5f0|                                       6f 99 1d|             o..|      crc: 0xb31d996f (valid) 0x5fd-0x601 (4)
+0x600|b3                                             |.               |
+     |                                               |                |  metaindex{}: 0x601-0x60e (13)
+     |                                               |                |    uncompressed{}: 0x601-0x609 (8)
+     |                                               |                |      trailer{}: 0x601-0x609 (8)
+     |                                               |                |        restarts[0:1]: 0x601-0x605 (4)
+0x600|   00 00 00 00                                 | ....           |          [0]: 0 restart 0x601-0x605 (4)
+0x600|               01 00 00 00                     |     ....       |        num_restarts: 1 0x605-0x609 (4)
+0x600|                           00                  |         .      |    compression: "none" (0x0) 0x609-0x60a (1)
+0x600|                              c0 f2 a1 b0      |          ....  |    crc: 0xb0a1f2c0 (valid) 0x60a-0x60e (4)
+     |                                               |                |  index{}: 0x60e-0x62a (28)
+     |                                               |                |    uncompressed{}: 0x60e-0x625 (23)
+     |                                               |                |      entries[0:1]: 0x60e-0x61d (15)
+     |                                               |                |        [0]{}: entry 0x60e-0x61d (15)
+0x600|                                          00   |              . |          shared_bytes: 0 0x60e-0x60f (1)
+0x600|                                             09|               .|          unshared_bytes: 9 0x60f-0x610 (1)
+0x610|03                                             |.               |          value_length: 3 0x610-0x611 (1)
+     |                                               |                |          key_delta{}: 0x611-0x61a (9)
+0x610|   73                                          | s              |            user_key: "s" 0x611-0x612 (1)
+0x610|      01                                       |  .             |            type: "value" (0x1) 0x612-0x613 (1)
+0x610|         ff ff ff ff ff ff ff                  |   .......      |            sequence_number: 72057594037927935 0x613-0x61a (7)
```

Comment:
Format as hex? but maybe decimal make sense if it's a sequence

Reply:
Typically sequence makes sense in decimal. They are sequential numbers. The only exception are these index  and metaindex sections; especially the example you highlighted, where the maximum possible unsigned integer `0xffffffffffffff` is chosen.

Reply:
Ok! keep as decimal. btw if 0xffffffffffffff has some special mening you can add description. For example the mp4 decoder does something like this for box size:
```go
const (
	boxSizeRestOfFile   = 0
	boxSizeUse64bitSize = 1
)

d.FieldU32("size", scalar.UintMapDescription{
	boxSizeRestOfFile:   "Rest of file",
	boxSizeUse64bitSize: "Use 64 bit size",
})
```
In your case you could probably just have `d.FieldU64("...", scalar.UintMapDescription{0xffffffffffffff: "blabla"})`

Reply:
@wader It's unclear to me if it has a special meaning. I only noticed so far that in the index and metaindex there are sometimes sequence numbers like that; it feels more like a hack though. The sequence numbers are typically used in the data section and there they have a well-defined meaning.

Reply:
Ok! then sounds like we should leave as it is